### ISSUE TITLE
Review and update puppetlabs-concat dependency

### ISSUE THREAD

**Issue by linuxmail** [voxpupuli/puppet-icinga2#351]
Hi,

just a question: must the Concat version older than 3.0.0 ? I had to change it, otherwise I wasn't able to upgrade Concat to the latest version for other modules.

cu denny

    { "name": "puppetlabs/concat", "version_requirement": ">= 2.0.1 < 3.0.0" },

**Comment by lbetz** [comment]
Should work but our tests are broken at the moment.

**Comment by lazyfrosch** [comment]
Also see #330

**Comment by cfoehrdes** [comment]
We are also experiencing lots of conflicts due to the restriction to this old concat release

**Comment by cristianbalan-mf** [comment]
@lazyfrosch, concat released a mix 3.0.0 and 4.0.0. While 4 can be seen as a port to P5, there's no reason for Icinga's module not to satisfy at least concat 3.
Concat 2 is just P3... :(
```
This double release adds new Puppet 4 data types. The 3.0.0 release is fully backwards compatible to existing Puppet 4 configurations and provides you with deprecation warnings for every argument that will not work as expected with the final 4.0.0 release. See the stdlib docs for an in-depth discussion of this.
If you're still running Puppet 3, remain on the latest puppetlabs-concat 2.x release for now...
```
